packages/typespec-autorest-canonical/README.md

```diff
@@ -0,0 +1,131 @@
+# @azure-tools/typespec-autorest-canonical
+
+TypeSpec library for emitting canonical version swagger
+
+## Install
+
+```bash
+npm install @azure-tools/typespec-autorest-canonical
+```
+
+## Emitter
+
+### Usage
+
+1. Via the command line
+
+```bash
+tsp compile . --emit=@azure-tools/typespec-autorest-canonical
+```
+
+2. Via the config
+
+```yaml
+emit:
+  - "@azure-tools/typespec-autorest-canonical"
+```
+
+### Emitter options
+
+#### `output-dir`
+
+**Type:** `string`
+
+Deprecated DO NOT USE. Use built-in emitter-output-dir instead
+
+#### `output-file`
+
+**Type:** `string`
```

Comment:
the doc for this should be updated or this option just removed all together

Reply:
update the doc